services/FileCreate.proto

```diff
@@ -43,7 +43,7 @@ After the file is created, the FileID for it can be found in the receipt, or rec
  */
 message FileCreateTransactionBody {
     Timestamp expirationTime = 2; // The time at which this file should expire (unless FileUpdateTransaction is used before then to extend its life)
-    KeyList keys = 3; // All these keys must sign to create or modify the file. Any one of them can sign to delete the file.
+    KeyList keys = 3; // All keys at the top level of a key list must sign to create or modify the file. Any one of the keys at the top level key list can sign to delete the file.
```

Comment:
Suggest to modify "All keys at the top level of a key list must sign to create or modify the file. Any one of the keys at the top level key list can sign to delete the file." to "The initial list of keys that can modify or delete the file."